__tests__/ExpensiMark-HTML-test.js

```diff
@@ -620,6 +620,48 @@ test('Test quotes markdown replacement and removing <br/> from <br/><pre> and </
     expect(parser.replace(testString)).toBe(resultString);
 });
 
```

Comment:
We have some failing tests, please update accordingly.

Reply:
I already updated tests after we decided to retain all blanks rows. `npm run test` is happy on my Mac and the GH check is also green. I'm a little bit confused about the failing tests you mentioned. Can you share more details please? Maybe I'm missing something

Reply:
@s77rt 
That's strange. Both [GH check](https://github.com/Expensify/expensify-common/actions/runs/4795896068/jobs/8531013931?pr=521) and my client are passed.

Hmm, did you checkout this branch to test on your client? If so, can you check the diff of `git diff HEAD~6`

Reply:
Oh so sorry. I was using an outsynced repo. The br difference is due to another change unrelated to this PR. All good now